packages/framework/tests/Unit/Model/Customer/CustomerUserUpdateDataFactoryTest.php

```diff
@@ -38,7 +39,9 @@ public function testGetAmendedCustomerUserUpdateDataByOrderWithoutChanges()
     {
         $customerUserUpdateUpdateDataFactory = $this->getCustomerUserUpdateDataFactory();
 
-        $customer = new Customer();
+        $customerData = new CustomerData();
```

Comment:
We should use `CustomerDataFactory` for creating `$customerData`. What do you say?

Reply:
I agree with that thought, but as it is Unit test you cannot get instance of factory there